README.md

```diff
@@ -14,7 +14,45 @@ currently) respectively.  `repobee-sanitizer` can remove or replace text by
 going through files, looking for certain `REPOBEE-SANITIZER-<type>`
 text-markers. The most simple usage consists of a `REPOBEE-SANITIZER-START` and
 a `REPOBEE-SANITIZER-END` marker, where content between these two markers will
-be removes, or "santitized".
+be removes, or "santitized" as it were.
+
+## Install
+Use RepoBee's plugin manager to install! 
+
+First, make sure that you have the latest version of `Repobee` installed,
+instructions can be found on the [Repobee website](https://repobee.org/).
+
+To install `repobee-sanitizer`, simply run the command `repobee plugin install` in your terminal, and this will open up a bullet-list where you can select sanitizer. 
+
+> If `repobee-sanitizer` does not show up when running the `sanitize` command. Run `repobee plugin list`, this will present a list with all the availiable plugins. If you cannot find `repobee-sanitizer` in the list, try updating `repobee` using repobee `repobee manage upgrade`.
+
+## Commands
+`repobee-sanitizer` supports two main commands:
+
+### File
+
+`repobee sanitize file <infile> <outfile>` performs the sanitization operation described below directly on a file `infile` and writes the output to `outfile` 
+
+The `--strip` flag can also be used to reverse the operation, instead removing all sanitizer syntax from the file.
+
+### Repo
+
+#### Repo root
+`repobee sanitize repo` performs the sanitization protocol on an entire repository. The directory you want to sanitize is either specified as the working directory `./` or it can be specified using the `--repo-root <path>` option.
+
```

Comment:
This should be the introduction to the section `sanitize repo` section, rather than a subsection (i.e. remove the `Repo roop` heading). It also gives a better overview if you start with an example command, and then add stuff to it. In short: explain by example!

````suggestion
`sanitize repo` performs the sanitization protocol on an entire repository, sanitizing the working directory of one branch and then committing the result to another branch. It's most basic usage looks like so.

```
$ repobee sanitize repo --target-branch main
```

Assuming that you're currently at the root of a Git repository on a branch other than `main`, this will sanitize the current branch and commit the result to `main`.
````

When that very basic usage has been explained, you can dive deeper into other options and concepts.